reverb/cc/platform/default/build_rules.bzl

```diff
@@ -107,20 +107,20 @@ def reverb_cc_proto_library(name, srcs = [], deps = [], **kwargs):
         name = "{}_static".format(name),
         srcs = gen_srcs,
         hdrs = gen_hdrs,
-        deps = depset([dep.replace(":", ":lib") + ".so" for dep in deps] + reverb_tf_deps()),
+        deps = depset([dep.replace(":", ":libxxx") + ".so" for dep in deps] + reverb_tf_deps()),
         alwayslink = 1,
         **kwargs
     )
     native.cc_binary(
-        name = "lib{}.so".format(name),
+        name = "libxxx{}.so".format(name),
         deps = ["{}_static".format(name)],
         linkshared = 1,
         **kwargs
     )
     native.cc_library(
         name = name,
         hdrs = gen_hdrs,
-        srcs = ["lib{}.so".format(name)],
```

Comment:
These `lib` -> `libxxx` changes was necessary to let my Bazel not complain about conflicting names. 
Inspired by https://github.com/bazelbuild/bazel/issues/4341#issuecomment-762576370.